aten/src/ATen/native/cpu/FlashAttentionKernel.cpp

```diff
@@ -367,25 +367,49 @@ void cpu_flash_attention(
 
   int64_t qSplitSize = q_split_size > qSize ? qSize : q_split_size;
   int64_t kvSplitSize = kv_split_size > kvSize ? kvSize : kv_split_size;
-  int64_t qSlice = (qSize - 1) / qSplitSize + 1;
-  int64_t kvSlice = (kvSize - 1) / kvSplitSize + 1;
-  int64_t kvTail = (kvSize - 1) % kvSplitSize + 1;
+  int64_t qSlice = (qSize + qSplitSize - 1) / qSplitSize;
+  int64_t kvSlice = (kvSize + kvSplitSize - 1) / kvSplitSize;
+  int64_t kvTail = kvSize % kvSplitSize;
   int64_t num_thread = at::get_num_threads();
 
   const auto dtype = query.scalar_type();
   const auto accumulate_dtype = toOpMathType(dtype);
 
   // Whether pack is needed
-  bool need_pack = with_pack && at::native::cpublas::need_pack(dtype);
+  bool need_pack = false;
```

Comment:
we force it to `false` now?

Reply:
No, I have added new check for `need_pack` below.